JitsiConference.js

```diff
@@ -72,6 +67,61 @@ function JitsiConference(options) {
 }
 
 /**
+ * Initializes the conference object properties
+ * @param options overrides this.options
+ */
+JitsiConference.prototype._init = function (options) {
+    if(!options)
+        options = {};
+    if(!this.options) {
+        this.options = options;
+    } else {
+        // Override config options
+        var config = options.config || {};
+        for(var key in config)
+            this.options.config[key] = config[key] || this.options.config[key];
+    }
```

Comment:
Agree. But currently we are not overriding anything from the options. I'll remove it for now.